setup.cfg

```diff
@@ -56,3 +56,5 @@ strict = True
 line_length = 127
 lines_between_types = 1
 lines_after_imports = 2
+known_first_part = build
+known_third_party = nox,pallets_sphinx_themes,pep517,pytest,setuptools,toml
```

Comment:
Do we need to explicitly declare this? isort hasn't been giving any issues.

Reply:
This makes running isort much faster because avoids the need to install these packages to discover, and makes explicit a list of used packages, so I'm personally strongly in favor it.

Reply:
Wouldn't `known_first_part` be enough then? `known_third_party` is just another extra place to maintain dependencies.

Reply:
Not it's not enough, see https://github.com/asottile/seed-isort-config#seed-isort-config for more details, @asottile can shed more light too

Reply:
Why doesn't isort have a config (possibly `first_part`) where I can say declare our packages? It feels very silly to me to add all modules we use in the config file when I can just say "threat anything other than `build` as an external module".

Reply:
You're free to raise the issue with isort, but that's not a thing today.